src/index.html

```diff
@@ -11,7 +11,127 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
+    <div class="wrapper">
+      <h1>HTML Form</h1>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+        <fieldset>
+          <legend>Personal information</legend>
+  
+          <label class="form-field">
+            Surname:
+            <input type="text" name="surname" 
+            autocomplete="off">
```

Comment:
put each new prop to the new line if there are more than 2-3 of them inside the element

```suggestion
            <input
              type="text"
              name="surname" 
              autocomplete="off"
            >
```